onehashai/onehash-cal
# Develop

Added Calendly integration to import user data from calendly

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

goto "/settings/import/calendly-import" and click on import button to authorize calendly and import user data

- Are there environment variables that should be set?
YES,The following environment variables need to be retrieved from https://developer.calendly.com/console/apps

NEXT_PUBLIC_CALENDLY_CLIENT_ID
CALENDLY_CLIENT_SECRET
CALENDLY_WEBHOOK_SIGNING_KEY
NEXT_PUBLIC_CALENDLY_REDIRECT_URI
NEXT_PUBLIC_CALENDLY_OAUTH_URL
NEXT_PUBLIC_CALENDLY_API_BASE_URL


## Mandatory Tasks

- [☑️ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
